File: c#/wypozyczalnia_samochodow_main/Customer_plik.cs
namespace Customer_plik; // by zainicjowac w Program.cs

public class Customer
{
    public string Imie { get; }
    public string Nazwisko { get; }
    public string Email { get; }

    public Customer(string imie, string nazwisko, string email)     // customer1, customer2 czy odwolanie później do pliku Rental_plik.cs do wyliczeń
    {
        Imie = imie;
        Nazwisko = nazwisko;
        Email = email;
    }

    public void klient()    // wyswietla info o kliencie
    {
        Console.WriteLine($"Klient: {Imie} {Nazwisko}, {Email}");
    }
}

File: python/Customer_plik.py
class Customer:
    def __init__(self, imie: str, nazwisko: str, email: str):
        self.imie = imie
        self.nazwisko = nazwisko
        self.email = email

    def klient(self):
        print(f"Klient: {self.imie} {self.nazwisko}, {self.email}")
